fluss-dist/pom.xml

```diff
@@ -30,41 +30,6 @@
 
     <dependencies>
 
-        <dependency>
-            <groupId>com.alibaba.fluss</groupId>
-            <artifactId>fluss-server</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-
-        <!-- start Fluss filesystem plugin -->
-        <dependency>
-            <groupId>com.alibaba.fluss</groupId>
-            <artifactId>fluss-fs-oss</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>com.alibaba.fluss</groupId>
-            <artifactId>fluss-fs-s3</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
-        <dependency>
```

Comment:
If remove these `dependency` in here,  fluss-dist wll be build before  `fluss-lakehouse-paimon`, then it'll throw 
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.6.0:single (opt) on project fluss-dist: Failed to create assembly: Error adding file to archive: /Users/luoyuxia/Project/fluss/fluss-dist/../fluss-lakehouse/fluss-lakehouse-paimon/target/fluss-lakehouse-paimon-0.6-SNAPSHOT.jar -> [Help 1]
` ```
while building..